refinery/data_set_manager/utils.py

```diff
@@ -522,8 +522,18 @@ def generate_solr_params_for_user(params, user):
         fq - filter query
      """
 
+    user = None
+    try:
+        user = User.objects.get(uuid=user_uuid)
+    except User.DoesNotExist:
+        pass
+
+    if user:
+        datasets = get_objects_for_user(user, "core.read_dataset")
+    else:
+        datasets = []
+
```

Comment:
A different approach would be:
```python
datasets = []
try:
    user = User.objects.get(uuid=user_uuid)
    datasets = get_objects_for_user(user, "core.read_dataset")
except User.DoesNotExist:
    pass
```
but my sense is that limiting the scope of the `try` is really important to us?